kokoro/build.sh

```diff
@@ -20,6 +20,13 @@ set -e
 # Code under repo is checked out to this directory.
 cd "${KOKORO_ARTIFACTS_DIR}/github/dataflux-pytorch"
 
+function setup_virtual_envs() {
+    echo Setting up Python virtual environment.
+    sudo apt install python3.8-venv
```

Comment:
Tried "python3-venv" but can't find candidate. Did some searches and seemed that we needed to specify the python version